ArduPlane/system.cpp

```diff
@@ -159,6 +159,11 @@ void Plane::init_ardupilot()
 // init cargo gripper
 #if GRIPPER_ENABLED == ENABLED
     g2.gripper.init();
+#endif
+
+    // init fence
+#if AC_FENCE == ENABLED
+    
```

Comment:
Well, that was unexciting....

Reply:
wow! Actually needs to init inside the guards.